<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="断言详情查看"
           tests="1"
           failures="1"
           errors="0"
           skipped="0"
           time="0.000">
  <properties />
            <testcase name="star" time="0.000" classname="Test.DataProviderSet_Assert">
                <failure type="java.lang.RuntimeException"
                  message="java.lang.NegativeArraySizeException"
              >
<![CDATA[java.lang.RuntimeException: java.lang.NegativeArraySizeException
  org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:161)
  org.testng.internal.Parameters.handleParameters(Parameters.java:429)
  org.testng.internal.Invoker.handleParameters(Invoker.java:1383)
  org.testng.internal.Invoker.createParameters(Invoker.java:1075)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1180)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
  org.testng.TestRunner.privateRun(TestRunner.java:767)
  org.testng.TestRunner.run(TestRunner.java:617)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
  org.testng.SuiteRunner.run(SuiteRunner.java:240)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
  org.testng.TestNG.run(TestNG.java:1057)
  configSelenium.ConfigSeleniumStart.dotestng(ConfigSeleniumStart.java:1061)
  configSelenium.ConfigSeleniumStart.main(ConfigSeleniumStart.java:215)
Caused by: java.lang.NegativeArraySizeException
  Test.DataProviderSet_Assert.getData(DataProviderSet_Assert.java:47)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
  org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:135)
  org.testng.internal.Parameters.handleParameters(Parameters.java:429)
  org.testng.internal.Invoker.handleParameters(Invoker.java:1383)
  org.testng.internal.Invoker.createParameters(Invoker.java:1075)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1180)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
  org.testng.TestRunner.privateRun(TestRunner.java:767)
  org.testng.TestRunner.run(TestRunner.java:617)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
  org.testng.SuiteRunner.run(SuiteRunner.java:240)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
  org.testng.TestNG.run(TestNG.java:1057)
  configSelenium.ConfigSeleniumStart.dotestng(ConfigSeleniumStart.java:1061)
  configSelenium.ConfigSeleniumStart.main(ConfigSeleniumStart.java:215)
]]>
      </failure>

          </testcase>
  
  
  
</testsuite>
